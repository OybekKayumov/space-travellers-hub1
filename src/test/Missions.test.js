import { render, screen } from '@testing-library/react';

const MissionsList = () => <>MissionsList</>;

test('should display the nav text Description', () => {
  render(<MissionsList />);
  screen.debug();
});
