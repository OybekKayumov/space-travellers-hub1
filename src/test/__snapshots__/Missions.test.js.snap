// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test renders the Missions component missions section renders a hardcoded object, passed as a prop 1`] = `
<div>
  <div
    style="width: 90%; margin-left: auto; margin-right: auto;"
  >
    <table
      class="table table-striped table-bordered table-hover"
    >
      <thead>
        <tr>
          <th>
            Missions
          </th>
          <th>
            Description
          </th>
          <th>
            Status
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</div>
`;
